### DISCUSSION TITLE
List of known working email services that work with Documenso

### DISCUSSION THREAD

**Discussion by Dvalin21** [documenso/documenso#1299] [General]
Hello, 

    I've been trying to get Documenso to allow me to signup, but none of the email services that I've tried to use will work. Google and Yahoo are the two that I've tried and I don't want to use resend as I would have to add my domain to it. Is there a list of working email services that will work. Or maybe I'm doing something wrong, could some please share there smtp setup let me know what email service they are using? This Documenso is via docker on a Synoloy NAS. I have other docker services that are able to reachout through smtp and send me an email so not sure why this wont. For google, it could just be that they are totally getting rid of the third party add access altogether soon. Also they left spaces in their passwords on purpose.

```
# [[SMTP]]
# OPTIONAL: Defines the transport to use for sending emails. Available options: smtp-auth (default) | smtp-api | mailchannels 
NEXT_PRIVATE_SMTP_TRANSPORT="smtp-auth"
NEXT_PRIVATE_SMTP_HOST="smtp.gmail.com"
NEXT_PRIVATE_SMTP_PORT="465"
NEXT_PRIVATE_SMTP_USERNAME="email address"
NEXT_PRIVATE_SMTP_PASSWORD="google password"
NEXT_PRIVATE_SMTP_SECURE="true"
```

**Comment by TSP-Dev** [comment]
aws ses

**Reply by Dvalin21** [reply]
Your joking right?

**Comment by ephraimduncan** [comment]
You can use Sendgrid, Resend or Mailgun

**Comment by Novapixel1010** [comment]
I am using forwardemail here is and example 
```

NEXT_PRIVATE_SMTP_TRANSPORT=smtp-auth
NEXT_PRIVATE_SMTP_HOST=smtp.forwardemail.net
NEXT_PRIVATE_SMTP_PORT=465
NEXT_PRIVATE_SMTP_USERNAME=no-reply@YOURDOMAIN.com
NEXT_PRIVATE_SMTP_PASSWORD=catch-all-password-REPLACE
NEXT_PRIVATE_SMTP_SECURE=true
NEXT_PRIVATE_SMTP_UNSAFE_IGNORE_TLS=false
NEXT_PRIVATE_SMTP_FROM_NAME=company name
NEXT_PRIVATE_SMTP_FROM_ADDRESS=no-reply@YOURDOMAIN.com
```
